applications/nrf_desktop/src/modules/Kconfig.dvfs_state_template

```diff
@@ -0,0 +1,52 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig DESKTOP_DVFS_STATE_$(dvfs_state)_ENABLE
+	bool "Handle DESKTOP_DVFS_STATE_$(dvfs_state)"
+	default y
+	depends on $(dvfs_dependencies)
+
+if DESKTOP_DVFS_STATE_$(dvfs_state)_ENABLE
+
+if $(dvfs_timeout) != 0
+
+config DESKTOP_DVFS_STATE_$(dvfs_state)_TIMEOUT_MS
+	int "timeout"
+	default $(dvfs_timeout)
+	range 0 65535
+	help
+	  Timeout in milliseconds specifying time after which DVFS request related to
+	  DESKTOP_DVFS_STATE_$(dvfs_state) times out.
+	  A request is automatically cleared on timeout.
+
+endif # $(dvfs_state)_TIMEOUT_MS != 0
+
+if $(dvfs_timeout) = 0
+
+config DESKTOP_DVFS_STATE_$(dvfs_state)_TIMEOUT_MS
+	int
+	default 0
+	range 0 0
```

Comment:
What is the reason to define it like that? Why not just leave one Kconfig option?